tests/main/snapshot-exclusions-dynamic/task.yaml

```diff
@@ -0,0 +1,132 @@
+summary: Check that the basic dynamic snapshot data exclusion functionality is ok
+
+environment:
+    # tar invoked when creating snapshots triggers OOM
+    SNAPD_NO_MEMORY_LIMIT: 1
+    USER_DATA_DIR: /root/snap/test-snap
+    SYSTEM_DATA_DIR: /var/snap/test-snap
+
+prepare: |
+    "$TESTSTOOLS"/snaps-state install-local test-snap
+
+restore: |
+    rm -f expected_options_entry actual_options_entry expected_restored_files actual_restored_files ./*_export.snapshot
```

Comment:
It is not needed to remove files from the test directory, it is automatically done during the gereric cleanup

Reply:
Did not know this, thanks